cbindings/blitzar_api.h

```diff
@@ -31,22 +31,22 @@ struct sxt_config {
   uint64_t num_precomputed_generators;
 };
 
-struct sxt_compressed_ristretto {
+struct sxt_ristretto255_compressed {
   // encodes an element of the ristretto255 group
   uint8_t ristretto_bytes[32];
 };
 
-struct sxt_scalar {
+struct sxt_curve25519_scalar {
   // encodes an element of the finite field modulo (2^252 + 27742317777372353535851937790883648493)
   uint8_t bytes[32];
 };
 
-struct sxt_transcript {
+struct sxt_curve25519_transcript {
```

Comment:
Transcript isn't unique to a curve. We can keep the sxt_transcript name here.

Reply:
Updated.